modules/ROOT/pages/integration/authorization.adoc

```diff
@@ -159,7 +159,8 @@ image::ig2/image14.png[Application authorization screen,333,303]
 
 agrirouter sends an HTTP 301 redirect to the browser, encoding the authorization result in a GET queue attached to the Redirect URL entered in the developers' application settings.
 
-The browser reacts in requesting this redirect URL which performs a GET request at the endpoint of the address.
+The browser reacts in requesting this redirect URL which performs a GET request at the endpoint of the address. The first redirect URL set comes into
+play when an authorize request does not provide a redirect URL (or provides an invalid one).
```

Comment:
"Please note that this behavior is DEPRECATED and we will require a VALID redirect URL to be provided to an authorize request in the future."
--> Please also note this in the "Parameters" section